### ISSUE TITLE
Slow checkout help

### ISSUE THREAD

**Issue by franc3sc0s92** [Automattic/jetpack#24945]
### Impacted plugin

Jetpack

### Steps to Reproduce

hi guys i have a problem that i can’t solve in any way. I use jetpack to connect the woocommerce app. I have all modules disabled but with jetpack active the checkout takes 8-9 seconds longer than normal.

I would like to remove everything jetpack does after hitting the ‘submit order’ button can you help me? thank you

The site is campettomarzi.it. you can also try to order

### A clear and concise description of what you expected to happen.

not jetpack in checkout

### What actually happened

jetpack slow in checkout

### Reproducibility

Consistent

### Severity

All

### Operating System

Windows, Linux

### Browser

Chrome/Chromium

**Comment by coder-karen** [comment]
Hi @franc3sc0s92 

Thanks for reaching out about this. The slow checkout you are experiencing is most likely related to Jetpack's ability to sync data, as we've received some similar reports. That's what will be happening after clicking the submit order button so it wouldn't be possible to remove that functionality. 

However, the good news is that the issue is expected to be resolved soon. Changes will be rolled-out starting later this week. It won't be to all sites at once so it's unclear when the changes will be noticeable on your site, however I've made a note for us to check back in when the changes should have been fully implemented to see if that has fixed the issue for you (internal ref: p7pQDF-6p3-p2#comment-21342).

I do notice whilst checking just now that Jetpack appears to be disconnected from your site though, so I'm not sure if you've disconnected it for other reasons.

**Comment by danjjohnson** [comment]
Hi @franc3sc0s92 

The changes Karen mentioned above have now been implemented, so if you can try reconnecting Jetpack on your site, you should find that there are no more speed issues with your checkout. You can find more info on reconnecting here:
https://jetpack.com/support/reconnecting-reinstalling-jetpack/

Let us know if you still have trouble after that.

**Comment by franc3sc0s92** [comment]
yes, I disconnected it due to the slowness. i will receive orders in
alternative way if there is no solution

Carmine Francesco Salvino

Il Lun 4 Lug 2022, 14:37 Karen Attfield ***@***.***> ha
scritto:

**Comment by coder-karen** [comment]
Hi @franc3sc0s92 

It looks like you might have missed our response on 16th Aug (it looks like you're responding via email so it might be that a follow-up was missed there). The changes that were mentioned have now been implemented, so you should be able to reconnect Jetpack and not have speed issues in the checkout. If you were still having speed issues after doing that, do let us know.
